storio-sqlite-annotations-processor-test/src/test/resources/BoxedTypesFields.java

```diff
@@ -0,0 +1,23 @@
+package com.pushtorefresh.storio.sqlite.annotations;
+
+@StorIOSQLiteType(table = "table")
+public class BoxedTypesFields {
+
+    @StorIOSQLiteColumn(name = "booleanField")
```

Comment:
Would be nice to make field and column names different so it would not looks like they have to be the same for other contributors/readers of the code!